control/autoware_autonomous_emergency_braking/src/node.cpp

```diff
@@ -827,7 +871,7 @@ void AEB::getClosestObjectsOnPath(
   const PointCloud::Ptr points_belonging_to_cluster_hulls, std::vector<ObjectData> & objects)
 {
   autoware::universe_utils::ScopedTimeTrack st(__func__, *time_keeper_);
-  // check if the predicted path has valid number of points
+  // check if the predicted path has a valid number of points
   if (ego_path.size() < 2 || ego_polys.empty() || points_belonging_to_cluster_hulls->empty()) {
```

Comment:
@danielsanchezaran,

Thank you for your valuable reviews and comments. I think it would be beneficial to add a new parameter for the collision detection method (polygons or arc coordinates). In my opinion, it would be better to implement this in a separate PR. After this PR, I can work on that :+1: